.codeclimate.yml

```diff
@@ -10,4 +10,4 @@ exclude_paths:
 - "**/*.md"
 - "Dockerfile"
 - "bin/fixme"
-- "tests/**"
+- "test/**"
```

Comment:
philosophical question: why do we exclude the tests, but still respond to :lipstick: comments?